interface/src/app/sample/sampleMocks.spec.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.mocks.sample', [])
```

Comment:
Apparently autocorrect was thinking `jaunt` looked better than `jsunit`. Too bad that makes no sense.